test/e2e/test.js

```diff
@@ -20,14 +20,13 @@ function fib(n) {
  * limitations under the License.
  */
 
-process.env.GCLOUD_INSIGHTS_LOGLEVEL=4;
-process.env.GCLOUD_DEBUG_ENABLED=true;
+process.env.GCLOUD_DEBUG_LOGLEVEL=3;
 
 var assert = require('assert');
-var util = require('util');
 var GoogleAuth = require('google-auth-library');
```

Comment:
Is there a reason why we need the dependency on google-auth-library here? We have corresponding functionality in diagnostics common which should already be used by this package.